crates/proof-of-sql/README.md

```diff
@@ -73,8 +73,13 @@ The "Hello World" example demonstrates generating and verifying a proof of the q
 #### Run
 
 ```bash
-cargo run --example hello_world
+cargo run --example hello_world --no-default-features --features="test" 
 ```
+> [!NOTE]
+> To run this example with the `blitzar` feature:
+> ```bash
+> cargo run --example hello_world --features="blitzar test"
+> ```
```

Comment:
Let's flip these. We prefer using blitzar since it is our GPU accelerator and makes everything faster.

````suggestion
cargo run --example hello_world 
```
> [!NOTE]
> To run this example without the `blitzar` (i.e. CPU only) feature:
> ```bash
> cargo run --example hello_world --no-default-features --features="rayon test"
> ```
````